lib/rules/empty-line-between-blocks.js

```diff
@@ -105,7 +105,8 @@ var findNearestReturnSass = function (parent, i) {
 module.exports = {
   'name': 'empty-line-between-blocks',
   'defaults': {
-    'include': true
+    'include': true,
+    'ignore-single-line-rulesets': true
```

Comment:
can we change this to `allow-single-line-rulesets` as per the latest comments in #282 ?